GitUI/CommandsDialogs/BrowseDialog/FormRecentReposSettings.Designer.cs

```diff
@@ -76,8 +76,8 @@ private void InitializeComponent()
             // 
             // flpnlControls
             // 
-            flpnlControls.Controls.Add(this.Ok);
             flpnlControls.Controls.Add(this.Abort);
+            flpnlControls.Controls.Add(this.Ok);
```

Comment:
The original order is more logical to me (should not make a difference). Revert, please.

Reply:
Just tried it, actually does make a difference
![grafik](https://user-images.githubusercontent.com/7032094/77247552-9dcf6200-6c32-11ea-83c0-c626541471f8.png)

Reply:
OK